databases/week2/part2.sql

```diff
@@ -0,0 +1,39 @@
+-- Part 2: School database
+-- Create a new database containing the following tables:
+
+CREATE DATABASE school;
+
+USE school;
+
+SET NAMES utf8mb4;
+
+-- Class: with the columns: id, name, begins (date), ends (date)
+CREATE TABLE `Class` (
+`id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+`name` VARCHAR(225) NOT NULL,
+`begins_date` DATETIME NOT NULL,
+`end_date` DATETIME NOT NULL,
```

Comment:
Data  type was supposed to be `date` not `datetime`. Pls modify as per the what was asked..? -- Class: with the columns: id, name, begins (date), ends (date)